### DISCUSSION TITLE
bottomShadowColor

### DISCUSSION THREAD

**Discussion by alexisasc** [ImaginativeShohag/Why-Not-Image-Carousel#118] [General]
Hello, Your library is wonderful

how can I use this property "bottomShadowColor" ?

since the entire app is white and if I cannot modify that app, the black line is noticeable

please enable that property

**Comment by ImaginativeShohag** [comment]
You can use the following properties:

```kotlin
carousel.showBottomShadow = true
carousel.bottomShadowAlpha = 0.6f // 0 to 1, 1 means 100%
carousel.bottomShadowHeight = 64.dpToPx(context) // px value of dp
```

In XML:

```xml
    app:showBottomShadow="true"
    app:bottomShadowAlpha="0.6"
    app:bottomShadowHeight="64dp"
```

If you want to remove the bottom shadow then use `carousel.showBottomShadow = false` in code or `app:showBottomShadow="false"` in XML.

**Comment by alexisasc** [comment]
Thank you for the quick response

This is what I have in xml:


   app:carouselType="SHOWCASE"
  app:imageScaleType="center"
  app:infiniteCarousel="true"
  app:scaleOnScroll="true"
  app:showBottomShadow="false"
  app:showTopShadow="false"
  
  
<img width="214" alt="123" src="https://github.com/ImaginativeShohag/Why-Not-Image-Carousel/assets/147612816/3f14d408-b206-490f-92a7-b93b22075e16">

###############################

If I place like this:
  
  
app:carouselType="SHOWCASE"
app:imageScaleType="center"
app:infiniteCarousel="true"
app:scaleOnScroll="false"
app:showBottomShadow="true"
app:showTopShadow="false"
app:bottomShadowHeight="0.5dp"

![WhatsApp Image 2024-06-11 at 17 56 56](https://github.com/ImaginativeShohag/Why-Not-Image-Carousel/assets/147612816/495e073f-91d4-43e3-aeaa-d5759d6b844a)

#############################################


but the app is all white  app:showBottomShadow="true"  , that line draws attention

**Comment by alexisasc** [comment]
When those properties are set to false, as the image shows, the images do not appear well in the carousel, they appear blank for a moment when I want to write in the chat

<img width="214" alt="123" src="https://github.com/ImaginativeShohag/Why-Not-Image-Carousel/assets/147612816/266cee1f-0ef1-44b7-837a-99e4dd35fed5">

**Comment by ImaginativeShohag** [comment]
I think there are other issues than this. BTW you can use `app:bottomShadowAlpha="0"` to hide the bottom shadow.

If that also does not solve your problem, share a minimum reproducible code, so that I can check the issue.

**Comment by alexisasc** [comment]
thank you very much, it worked.

app:bottomShadowAlpha="0"


very thankful
